src/components/shared/button.tsx

```diff
@@ -1,11 +1,11 @@
 import React from "react";
 
-const PrimaryButton = props => {
-  const { children, onClick, disabled } = props;
+const Button = props => {
+  const { children, onClick, disabled, className } = props;
```

Comment:
I think we should use styled components instead of className. add an optional boolean attribute `primary`. So it'll look like this:

```
const Button = styled.button`
  color: ${ props => props.primary ? "blue" : "default" }
`
<Button onClick={() => console.log("foobar")} disabled={false}>Not primary</Button>
<Button primary onClick={() => console.log("foobar")} disabled={false}>Primary</Button>
```

Reply:
@EdmundMai Ok, agree